.circleci/config.yml

```diff
@@ -45,13 +45,13 @@ workflows:
 
       # deploy to aws installations (only tags)
       - architect/push-to-app-collection:
```

Comment:
I'd just remove this job, to avoid the issue we had last time. (I mean from this branch only)

Reply:
done